telethon/tl/custom/message.py

```diff
@@ -617,19 +617,21 @@ def get_entities_text(self, cls=None):
                 >>> for _, inner_text in m.get_entities_text(MessageEntityCode):
                 >>>     print(inner_text)
         """
+        entities_list = self.original_message.entities
+        
         if not self.original_message.entities:
             return []
 
         if cls and self.original_message.entities:
+            entities_list = [c for c in self.original_message.entities if isinstance(c, cls)]
```

Comment:
This line is too long, please split the comprehension accross more than one line or use the shorter name:
```python
entities_list = [c for c in entities_list if isinstance(c, cls)]
```